### PR TITLE
#455 JSDoc types 

### PR DESCRIPTION
**Pull request by Nazanin1369**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2652,7 +2652,7 @@ Unit testing helps maintain clean code, as such I included some of my recommenda
           /**
            * @name logError
            * @desc Logs errors
-           * @param {String} msg Message to log
+           * @param {String} msg - Message to log.
            * @returns {String}
```
Comment (mattgrande) [inline README.md]: These should probably be changed to `string` (lowercase), to match what is said below.

(Also, I'm not a fan of using the hyphen in the `@param` description, but that's entirely personal preference, and I'm not bothered either way; I'm interested to hear the opinions of others on this)

Comment (Nazanin1369) [inline README.md]: Changed it. 

This is what is officially in JSDoc documentation. Will welcome any other opinions on this.

**Comment by johnpapa** [comment]
the corrections are good. but i dont want to add a lot of detail about jsdocs here in the ng style guide. can you revise please?

**Comment by Nazanin1369** [comment]
Sure. Will remove the types explanation part

**Comment by Nazanin1369** [comment]
@johnpapa Removed the explanations. Only corrections are there.

**Comment by johnpapa** [comment]
thanks, can you adjust to the new folder so we dont have merge conflicts? reopen when ready

**Pull request closed without merging**